install.sh

```diff
@@ -90,7 +112,8 @@ function install_packages {
     install_yum
 }
 
-if [ -n "${GDB}" ]; then
+
+if [ "${BACKEND_GDB}" -eq 1 ]; then
```

Comment:
The `$BACKEND_` variables are set at the beginning of the script (lines 19-22), the default values check whether `$GDB`/`$LLDB` are non-empty.